style.css

```diff
@@ -102,3 +137,246 @@ span {
   color: #505f79;
   width: 1.2rem;
 }
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  gap: 2rem;
+  margin-left: 1rem;
+}
```

Comment:
- [ ] As instructed according to the project requirement, kindly use CSS grid property to display the contents of this container. 👍